nextcord/application_command.py

```diff
@@ -2962,3 +3096,76 @@ def get_roles_from_interaction(state: ConnectionState, interaction: Interaction)
             ret.append(role)
 
     return ret
+
+
+def unpack_annotated(given_anno, resolve_list: list = []) -> type:
+    origin = typing.get_origin(given_anno)
+    if origin is Annotated:
+        located_annotation = MISSING
+        arg_list = typing.get_args(given_anno)
+        for i in range(1, len(arg_list)):
+            if arg_list[i] in resolve_list:
+                located_annotation = arg_list[i]
```

Comment:
could python `enumerate` be used here?
```suggestion
        for i, arg in enumerate(arg_list, start=1):
            if arg in resolve_list:
                located_annotation = arg
```
`start=1` as i assume your range starting with one means you want to skip the first element

if you do not need `i` (seemingly you dont)
```suggestion
        for arg in arg_list[1:]:
            if arg in resolve_list:
                located_annotation = arg
```
if you do need the first element
```suggestion
        for arg in arg_list:
            if arg in resolve_list:
                located_annotation = arg
```